eval/context.go

```diff
@@ -219,31 +221,34 @@ func (c *Context) WithOAuth2(o []*config.OAuth2AC) *Context {
 	return c
 }
 
-// WithSAML initially setup the lib.FnSamlSsoUrl function.
+// WithSAML initially setup the saml configuration.
```

Comment:
With only initializing `c.saml` left, is this still a good place? Same with `WithOAuth2()`